mainframer.sh

```diff
@@ -31,6 +31,13 @@ COMMON_IGNORE_FILE="$CONFIG_DIR/ignore"
 LOCAL_IGNORE_FILE="$CONFIG_DIR/localignore"
 REMOTE_IGNORE_FILE="$CONFIG_DIR/remoteignore"
 
+ HOURS_TAG="hours"
+ MINUTES_TAG="minutes"
+ SECONDS_TAG="seconds"
+ HOURS_FORMAT="%02d"
+ MINUTES_FORMAT="%02d"
+ SECONDS_FORMAT="%02d"
```

Comment:
What would you prefer? I mean omitting minutes is clear but should seconds be just one digit in case they are 1-9?